clients/js/src/ChromaClient.ts

```diff
@@ -168,6 +168,7 @@ export class ChromaClient {
         this.database,
         {
           name,
+          configuration: null, //TODO: Configuration type in JavaScript
```

Comment:
Is our plan to just not support this in JS?

Reply:
We will support it in JS, with a proper object, but for the time being we don't. Since there's nothing user-facing here yet anyway, we have time to add proper JS support in a forthcoming PR as part of the JS client parity work in this milestone.